pylhc/constants/calibration.py

```diff
@@ -0,0 +1,133 @@
+"""
+Constants: BPM Calibration
+-------------------------
```

Comment:
The title underline is too short here, needs one more.
This will crash the documentation build, because sphinx is like that